browser/ui/browser_commands.cc

```diff
@@ -942,8 +944,14 @@ void NewSplitViewForTab(Browser* browser, std::optional<tabs::TabHandle> tab) {
                                 ? model->IndexOfFirstNonPinnedTab()
                                 : tab_index + 1;
 
-  chrome::AddTabAt(browser, GURL("chrome://newtab"), new_tab_index,
-                   /*foreground*/ true);
+  if (!url.is_valid()) {
+    chrome::AddTabAt(browser, GURL("chrome://newtab"), new_tab_index,
+                     /*foreground*/ true);
+  } else {
+    chrome::AddTabAt(browser, url, new_tab_index,
```

Comment:
perhaps we want this only for http/https?

Reply:
Aha! That's a good question. If we want split view to be loaded only for HTTP or HTTPS,
 then we should also handle it in the context menu by not showing the menu item, right?

Reply:
@sangwoo108 Could you kindly confirm whether we want to restrict it to the HTTP/HTTPS scheme or not?

Reply:
I don't think we need to, as it's almost like opening new inactive tab. We don't limit the scheme in that case?